Tasks/BashV3/bash.ts

```diff
@@ -72,7 +73,20 @@ async function run() {
                 targetFilePath = input_filePath;
             }
 
-            contents = `. '${targetFilePath.replace("'", "'\\''")}' ${input_arguments}`.trim();
+            if (old_source_behavior) {
+                contents = `. '${targetFilePath.replace("'", "'\\''")}' ${input_arguments}`.trim();
+            } else {
+                // Check if executable bit is set
+                let stats: fs.Stats = tl.stats(input_filePath);
```

Comment:
This is where the mistake was last time - in the last iteration of this PR, I was doing a stats call on `targetFilePath` instead of `input_filePath`. This is a problem because `targetFilePath` gets some modifications on just windows so that it will work with bash, that cause stats to fail.